### PR TITLE
[Small cleanups] Remove libra from network checker test and fix key manager smoke test. 

### PR DESCRIPTION
**Pull request by JoshLind**
## Motivation

This PR offers two commits for simple code cleanups/improvements for tests:

1. First, remove "libra" from the network checker test and remove the redundant assertion.
2. Second, reduce the aggression of the key manager smoke test by adding a small time buffer.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

All tests pass locally.

## Related PRs

None.

### CODE REVIEW

File: testsuite/smoke-test/src/key_manager.rs [hunk 1]
```diff
@@ -54,6 +54,9 @@ fn test_key_manager_consensus_rotation() {
         ChainId::test(),
     );
 
+    // Add some time padding to ensure the libra timestamp increases on-chain
+    sleep(Duration::from_secs(10));
```
Comment (gregnazario) [inline testsuite/smoke-test/src/key_manager.rs:58]: :(. Isn't 10 seconds pretty long?

**Review by gregnazario — APPROVED** [review]
We can stick to 10 seconds, but let's try to improve upon it later

File: testsuite/smoke-test/src/key_manager.rs (hunk 1 above)
Comment (JoshLind) [reply testsuite/smoke-test/src/key_manager.rs:58]: Yeah, this is a beefy buffer.. I was trying to be conservative to reduce the flakes around the timestamp. As discussed offline, let's leave it as-is for now, and I'll monitor flakiness.... if things improve, I can reduce the timeout 😄

**Comment by JoshLind** [comment]
Thanks @gregnazario!

/land

**Comment by JoshLind** [comment]
Closing this out given the merge conflict and the fact that the repo rename has borked my ability to rebase this PR. Will open a new PR with only the key manager fix. 'cc @gregnazario

**Comment by gregnazario** [comment]
> Closing this out given the merge conflict and the fact that the repo rename has borked my ability to rebase this PR. Will open a new PR with only the key manager fix. 'cc @gregnazario

You should be able to do it all and use the same PR

**Pull request closed without merging**